hs_gabbs_web_app/utils.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+import os
+import logging
+
+from django.conf import settings
+
+from hs_core.hydroshare.utils import get_resource_by_shortkey
+
+
+logger = logging.getLogger(__name__)
+
+
+def push_res_to_geohub(url, user, shortkey):
```

Comment:
I'm reviewing what's available through the rest api and found this.  https://github.com/hydroshare/hydroshare/blob/develop/hs_core/views/utils.py#L758

It looks pretty easy to extend functionality for copying files and then this copy functionality would also be available through the rest api.

Reply:
I'm looking into this because @dtarb wanted me to see if we can replicate the copying of folders between federated irods that you're doing here through the rest api.  It currently doesn't exist, but I think it wouldn't be much work to extend this method to do a copy (provide a flag that indicates a copy or a move and check it to call the copy or move function on the istorage.

Dave wants to expose as much as we can through a rest api for 3rd parties to integrate with hydroshare, so even though we don't have a use case for having it in the rest api, someone 3rd party might.